agent/address.h

```diff
@@ -60,6 +60,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <netdb.h>
```

Comment:
I compile on ubuntu 16.04 and the addrinfo struct is defined in netdb.h on my platforn. According to the man page this is correct.